File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        String[] alpha = {"a","b","c","d","e","f","g","h","i","j"};
        String str = new String(age + "");
        String strArr[] = str.split("");
        
        for (int j = 0; j < strArr.length; j++){
            for(int i = 0; i < alpha.length; i++){
                if (strArr[j].equals(String.valueOf(i))){
                    answer += alpha[i];
                }
            
            }
                
                
        }
        
        
       
        
        return answer;
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    answer = ''
    n = {'a' : 0, 'b' :1, 'c':2, 'd' : 3 , 'e':4,'f' : 5, 'g' : 6, 'h' : 7, 'i' : 8, 'j':9}
    age = list(str(age))
    
    for a in age:
        for i , j in n.items():
            if j == int(a):
                answer += i
    
            
        
        
    return answer